lib/models/todo.dart

```diff
@@ -1,8 +1,12 @@
 import 'package:flutter/foundation.dart';
 
 class Todo {
```

Comment:
kindly describe the purpose of this class, as well as the fields as discussed in the call.

here's an example
![image](https://user-images.githubusercontent.com/41381713/106359219-34d57200-634c-11eb-9d2d-12547116c5f1.png)